src/lib.rs

```diff
@@ -481,10 +481,7 @@ where
     ) -> Result<(), Error<E>> {
         // Request measurement
         self.send_command(Command::Measure {
-            low_power: match mode {
-                PowerMode::LowPower => true,
-                PowerMode::NormalMode => false,
-            },
+            power_mode: mode,
```

Comment:
If you name the parameter `power_mode` as well, you don't need to use `: mode`.

Reply:
Fixed.